packages/client/components/DashboardAvatars/DashboardAvatars.tsx

```diff
@@ -2,56 +2,34 @@ import styled from '@emotion/styled'
 import graphql from 'babel-plugin-relay/macro'
 import React, {useMemo} from 'react'
 import {commitLocalUpdate, createFragmentContainer} from 'react-relay'
-import {Breakpoint, ElementHeight, ElementWidth} from '~/types/constEnums'
-import fromTeamMemberId from '~/utils/relay/fromTeamMemberId'
+import {ElementWidth} from '~/types/constEnums'
 import useAtmosphere from '../../hooks/useAtmosphere'
-import useBreakpoint from '../../hooks/useBreakpoint'
 import useMutationProps from '../../hooks/useMutationProps'
 import ToggleTeamDrawerMutation from '../../mutations/ToggleTeamDrawerMutation'
 import {PALETTE} from '../../styles/paletteV3'
 import {DashboardAvatars_team} from '../../__generated__/DashboardAvatars_team.graphql'
-import ErrorBoundary from '../ErrorBoundary'
+import AvatarList, {sizeToHeightBump} from '../AvatarList'
 import PlainButton from '../PlainButton/PlainButton'
-import DashboardAvatar from './DashboardAvatar'
 
-const AvatarsList = styled('div')<{isDesktop: boolean}>(({isDesktop}) => ({
+const Wrapper = styled('div')({
   display: 'flex',
   flexDirection: 'column',
-  marginRight: 6,
```

Comment:
`marginRight` & `left` usage looks deprecated (tested on desktop and mobile view)